### PR TITLE
Hotfix for SP NameID getters

### PR DESCRIPTION
**Pull request by nikosev**

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<ruleset name="SimpleSAMLphp SAML2 ruleset">
+    <config name="ignore_warnings_on_exit" value="1"/>
+
```
Comment (NicolasLiampotis) [inline phpcs.xml:4]: do we need this file?

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Do we need phpcs.xml? Other than that LTGM

File: phpcs.xml (hunk 1 above)
Comment (nikosev) [reply phpcs.xml:4]: This file is included in the core SSP modules (eg. [discopower](https://github.com/simplesamlphp/simplesamlphp-module-discopower/blob/master/phpcs.xml)), so I added this file to RCIAM modules in order to keep up with the code style of SimpleSAMLphp

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**